subprojects/model-core/src/testFixtures/groovy/org/gradle/internal/reflect/validation/ValidationMessageChecker.groovy

```diff
@@ -50,6 +54,9 @@ trait ValidationMessageChecker {
             .render()
     }
 
+    @ValidationTestFor(
+        ValidationProblemId.IGNORED_ANNOTATIONS_ON_METHOD
```

Comment:
I suppose you added this annotation so it is easy to find the test fixture for a given type?

Reply:
Absolutely, it's a bit of abuse of the annotation but it was useful for me to track everything.